str/reviewer_runner.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+"""
+This script will output REViewer svg based on inputs: one/multiple CPG IDs and one locus, as defined in the variant catalog. 
+analysis-runner --access-level test --dataset hgdp --description "reviewer" --output-dir 'str/410_sgd_loci/reviewer' reviewer_runner.py --catalog=gs://cpg-hgdp-test/str/410_sgdp_loci/catalogs/eh_catalog_hg38_backbone_trimmed_0_based.json --locus=chr3-67969584-67969611-AAC --input-dir=gs://cpg-hgdp-test/str/sensitivity-analysis/eh/trimmed_coordinates_0_based_hg38_backbone CPG265538		
+
+"""
+import os
+import click
+
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import output_path
+from cpg_workflows.batch import get_batch
+
+REF_FASTA = 'gs://cpg-common-main/references/hg38/v0/Homo_sapiens_assembly38.fasta'
```

Comment:
Can you pull this from the config?

Reply:
The masked fasta can be pulled with: ref_fasta = reference_path('broad/ref_fasta')

Reply:
Ah okay, I won't push it for this PR, but could you add an "ref_fasta_unmasked" to https://github.com/populationgenomics/references when you get a moment?

Reply:
yes